packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -180,53 +266,11 @@ describe('Component > InteractionLayer', function () {
             releasePointerCapture: sinon.stub()
           }
         }
+        const finishedMark = Object.assign({}, mockMark, { finished: true })
+        wrapper.setProps({ activeMark: finishedMark })
         wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
-        wrapper.simulate('pointermove', fakeEvent)
-        expect(fakeEvent.target.setPointerCapture.withArgs('fakePointer')).to.have.been.calledOnce()
-      })
-
-      describe('onPointerDown with a TranscriptionLine mark already in progress', function () {
-        it('should call the handlePointerDown function', function () {
-          const mockDrawingTask = DrawingTask.TaskModel.create({
-            activeToolIndex: 0,
-            instruction: 'draw a mark',
-            taskKey: 'T0',
-            tools: [
-              {
-                marks: {},
-                max: 2,
-                toolComponent: TranscriptionLine,
-                type: 'transcriptionLine'
-              }
-            ],
-            type: 'drawing'
-          })
-          activeTool = mockDrawingTask.activeTool
-          sinon.stub(activeTool, 'createMark').callsFake(() => mockMark)
-          sinon.stub(activeTool, 'handlePointerDown').callsFake(() => mockMark)
-
-          wrapper = shallow(
-            <InteractionLayer
-              activeMark={mockMark}
-              activeTool={activeTool}
-              frame={2}
-              height={400}
-              width={600}
-            />
-          )
-
-          const fakeEvent = {
-            pointerId: 'fakePointer',
-            type: 'pointer',
-            target: {
-              setPointerCapture: sinon.stub(),
-              releasePointerCapture: sinon.stub()
-            }
-          }
-          wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
-          wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
-          expect(activeTool.handlePointerDown).to.have.been.calledOnce()
-        })
+        wrapper.find(DrawingCanvas).simulate('pointerup', fakeEvent)
+        expect(fakeEvent.target.releasePointerCapture.withArgs('fakePointer')).to.have.been.calledOnce()
```

Comment:
Since writing the original code, I've discovered that the browser releases pointer capture for you, when you end a pointer interaction. I think we can get rid of `releasePointerCapture` from our code. That might also explain invalid pointer errors in Sentry, where our code is trying to release a pointer that's already been released by the browser.

>The following example shows a pointer capture being released (when a pointercancel event occurs. The browser does this automatically when a pointerup or pointercancel event occurs.

https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events#pointer_capture

Reply:
I'd prefer to keep this scoped to the focus bug fix. I can track this suggestion in an issue.